io_scene_nif/modules/nif_import/property/texture/__init__.py

```diff
@@ -204,11 +204,11 @@ def create_texture_slot(self, b_mat, n_tex_desc):
 
     def link_diffuse_node(self, b_texture_node):
         self.diffuse_texture = b_texture_node
-        b_texture_node.label = "Diffuse"
+        b_texture_node.label = "Base"
```

Comment:
We should probably extract these to constants. 
Probably need to make them accessible via the ui|?

Reply:
It's exposed as a text entry in the node editor, we could probably turn it into a dropdown but I'm not sure.

Reply:
Yeah something to consider from UX pov.